test/wallet-account.test.js

```diff
@@ -243,6 +243,71 @@ it('requests transaction signing automatically when function call has attached d
     expect(transactions).toHaveLength(1);
 });
 
+it('requests transaction signing with 2fa access key', async () => {
+    let localKeyPair = nearApi.KeyPair.fromRandom('ed25519');
+    let walletKeyPair = nearApi.KeyPair.fromRandom('ed25519');
+    setupWalletConnectionForSigning({
+        allKeys: [ walletKeyPair.publicKey.toString() ],
+        accountAccessKeys: [{
+            access_key: {
+                nonce: 1,
+                permission: {
+                    FunctionCall: {
+                        allowance: '1000000000',
+                        receiver_id: 'signer.near',
+                        method_names: ['add_request_and_confirm']
```

Comment:
for readability: it would be helpful to pull the string out into a constant in src/wallet-account.ts and reference that constant here, so it's clear that the method name has to be something specific.

Reply:
I think might be better to keep it in tests (even if extracting into constant), as this serves similar purpose as double entry accounting (avoid accidental rename in this case).

Reply:
ok